assemblies/pentaho-solutions/src/main/resources/pentaho-solutions/system/system.properties

```diff
@@ -77,3 +77,8 @@ cors-requests-allow-credentials=true
 
 # Exposed response HTTP headers for all cross-origin requests:
 cors-requests-exposed-headers=
+
+# ssrf protection  should be enabled by default for all requests to prevent server side request forgery attacks
+# Set to true to enable SSRF protection for the server.
+# ssrf protection is applied only if the ssrf-protection-enabled  is set to true
```

Comment:
I suggest the following rewording, in part for consistency with CSRF and CORS corresponding properties:
```properties
## SSRF ##

# Enables SSRF (Server-Side Request Forgery) protection for the server.
#
# Set to `true` to enable SSRF protection for the server.
#
# When protection is enabled, the server will only accept HTTP requests whose Host header
# is compatible with one of the server's fully-qualified URLs, as configured in `fully-qualified-server-url` 
# and `alternate-fully-qualified-server-urls`, in `server.properties`. 
# This applies to all endpoints exposed by the server.
# 
# *ATTENTION*:
# Setting this option to false opens the server and all of its plugins to SSRF attacks.
```

Additionally, given that SSRF is the foundation for CORS/CSRF protection, I propose placing the SSRF section before the CSRF one.

Reply:
modified documentation